sdk/openai/azure-ai-openai-assistants/src/samples/java/com/azure/ai/openai/assistants/ReadmeSamples.java

```diff
@@ -153,7 +153,7 @@ public void simpleMathAssistantOperations() throws InterruptedException {
     public void simpleRetrievalOperation() throws InterruptedException {
         String apiKey = Configuration.getGlobalConfiguration().get("NON_AZURE_OPENAI_KEY");
         String deploymentOrModelId = "gpt-4-1106-preview";
-        String fileName = "retrieval_sample_java_aoai_sdk.txt";
```

Comment:
keyword `aoai` is unknown word to spelling check and it only used by SDK team we should try to avoid using it.